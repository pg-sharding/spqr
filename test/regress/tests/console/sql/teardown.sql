DROP KEY RANGE ALL;
DROP SHARDING RULE ALL;
