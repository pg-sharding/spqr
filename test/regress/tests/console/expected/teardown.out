
		SPQR router admin console
	Here you can configure your routing rules
------------------------------------------------
	You can find documentation here 
https://github.com/pg-sharding/spqr/tree/master/docs

DROP KEY RANGE ALL;
 drop key range 
----------------
(0 rows)

